packages/snap-store-mobx/src/Finder/FinderStore.ts

```diff
@@ -6,16 +6,17 @@ import { PaginationStore } from '../Search/Stores';
 import { StorageStore, StorageType } from '../Storage/StorageStore';
 import { SelectionStore } from './Stores';
 import type { FinderStoreConfig, StoreServices, SelectedSelection, FinderStoreState } from '../types';
+import { UrlManager } from '@searchspring/snap-url-manager';
 
 export class FinderStore extends AbstractStore {
 	services: StoreServices;
-	config: FinderStoreConfig;
-	data: SearchResponseModel & { meta: MetaResponseModel };
+	config!: FinderStoreConfig;
```

Comment:
I refactored all class properties to use `!` where needed. I believe these are needed since it doesn't make much sense to make these optional and in cases like this one, config is set on the AbstractStore so it complains in the FinderStore because it's not assigned in it's constructor.

Reply:
Ok, this feels a bit awkward still, but I cannot think of any alternatives.